assisted-events-scrape/events_scrape/events_scrape.py

```diff
@@ -38,225 +32,77 @@
 
 
 class ScrapeEvents:
-    def __init__(self, inventory_url: str, offline_token: str, index: str, es_server: str, es_user: str,
-                 es_pass: str, backup_destination: str, is_sentry_enabled: bool, max_idle_minutes: int,
-                 errors_before_restart: int):
-
-        self.latest_update = datetime.now()
-        self.inventory_url = inventory_url
-        self.client = ClientFactory.create_client(url=self.inventory_url, offline_token=offline_token)
-        self.index = index
-        http_auth = None
-
-        if es_user:
-            http_auth = (es_user, es_pass)
+    def __init__(self, config: dict):
+        if config["backup_destination"] and not os.path.exists(config["backup_destination"]):
+            os.makedirs(config["backup_destination"])
 
-        self.es = elasticsearch.Elasticsearch(es_server, http_auth=http_auth)
-        self.backup_destination = backup_destination
-        if self.backup_destination and not os.path.exists(self.backup_destination):
-            os.makedirs(self.backup_destination)
+        self.client = ClientFactory.create_client(
+            url=config["inventory_url"],
+            offline_token=config["offline_token"])
```

Comment:
max line length is set to 120

```suggestion
        self.client = ClientFactory.create_client(url=config["inventory_url"], offline_token=config["offline_token"])
```